### ISSUE TITLE
Database migration cleanup

### ISSUE THREAD

**Issue by xtine** [18F/fec-cms#743]
I think it would be helpful if could assess the current state of DB migrations for the CMS. Maybe decide on squashing migrations if we want a clean slate to prevent past references that don't exist?

We should also look at the `django_migrations` table on both staging and prod if they match up with the migrations we actually do have in the repo.

Tagging @ccostino and @LindsayYoung to help in this effort.

**Comment by ccostino** [comment]
Agreed, @xtine!  I started looking into this a bit and it doesn't look too bad, but I didn't want to just do it either.  I'm more than happy to work with you and anyone else interested on this. :-)

**Comment by xtine** [comment]
@ccostino: let's plan a pairing session, I would like to get more confident with these DB problems.

**Comment by ccostino** [comment]
Perfect!  I'll find time on our calendars. :-)

**Comment by ccostino** [comment]
@noahmanger:  After reviewing more of the squashing mechanisms and what's involved with them, @xtine and I feel fairly confident that we should be able to move forward with this next sprint.  With the work on the scraping and importing being almost complete in this sprint, we should be a in a good spot where there isn't as much churn in the database with new models being added and other adjustments being made.

Furthermore, if we set things up to get this working at the start of next sprint, we can coordinate with everyone and actually do the squashing and initial testing while we're all in town together.  This would be a great learning and knowledge sharing opportunity for the FEC folks.

If this sounds good to you, we can start working out some specific dates and the people that should be involved and plan from there!  Please let us know if you have any questions or if anything is unclear.

**Comment by noahmanger** [comment]
Sounds good to me. Thanks!

**Comment by ccostino** [comment]
@xtine and I tried to go through the exercise of squashing the migrations with folks at the CMS today and made some progress, but there's more work to be done.  The squashed migration required one manual edit after the fact (which the migration framework indicated to us was the case!), and seems to work fine if you are working in an existing database.

However, when we tested the squashed migrations on a new database, we ran into an issue that we're currently unsure of.  The same issue occurs in Travis as well since each build starts with an empty database.  The current work is found in PR #826.
